components/calendly_v2/sources/new-event-scheduled/new-event-scheduled.mjs

```diff
@@ -0,0 +1,69 @@
+import app from "../../calendly_v2.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  key: "calendly_v2-new-event-scheduled",
+  name: "New Event Scheduled",
+  description: "Emit new event when a event is scheduled.",
```

Comment:
```suggestion
  description: "Emit a new event when an event is scheduled.",
```

Reply:
Changed "Emit new event" to "Emit a new event" to follow Google's style guide on using articles.